morpheushelper/cogs/joined_info.py

```diff
@@ -0,0 +1,38 @@
+from datetime import datetime
+
+from PyDrocsid.translations import translations
+from discord import Member
+from discord.ext import commands
+from discord.ext.commands import Cog, Context, Bot, guild_only
+
+from enum import Enum
+
+
+class Dates(Enum):
+    WEEK = 7
+    MONTH = 30
+    YEAR = 365
+
+
+def date_diff_to_str(date_diff):
+    if date_diff < Dates.WEEK.value:
+        return translations.joined_days
+    if date_diff < Dates.MONTH.value:
+        return translations.f_joined_weeks(date_diff // Dates.WEEK.value)
+    if date_diff < Dates.YEAR.value:
+        return translations.f_joined_months(date_diff // Dates.MONTH.value)
+    return translations.f_joined_years(date_diff // Dates.YEAR.value)
+
+
+class JoinedInfoCog(Cog):
+    def __init__(self, bot: Bot):
+        self.bot = bot
+
+    @commands.command()
+    @guild_only()
+    async def joined(self, ctx: Context, user: Member):
+        """
+        Returns the date on which the user joined the server
+        """
+        date_diff = (datetime.today() - user.joined_at).days
```

Comment:
which join date is this and shouldn't we use the first join after the last automatic kick